### PR TITLE
[Api][Web][SIMS #879] Student Notes

### PR DESCRIPTION
**Pull request by dheepak-aot**
- [ ] Organize "most recent" notes to be at the top of the page, for both institution and student notes
- [ ] Create student notes Vue and route. As the APIs already exist.

### CODE REVIEW

File: sources/packages/web/src/views/aest/student/StudentNotes.vue [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      required: true,
+    },
+  },
+  setup(props: any) {
+    const notes = ref();
+    const filteredNoteType = ref("");
+    const { dateOnlyLongString } = useFormatters();
+    const toast = useToastMessage();
+
+    const filterNotes = async (noteType?: StudentNoteType | string) => {
```
Comment (ann-aot) [inline sources/packages/web/src/views/aest/student/StudentNotes.vue:57]: why it or `string`?

**Review by ann-aot — COMMENTED** [review]
lGTM @dheepak-aot . just a minor comment

**Review by abschwenker — APPROVED** [review]

**Review by guru-aot — APPROVED** [review]
LGTM. good work

**Review by ann-aot — APPROVED** [review]
Good Job with Student Notes @dheepak-aot Thanks for addressing the comments 👍

**Pull request merged**